templates/aurora.cfn.yml

```diff
@@ -74,22 +76,127 @@ Parameters:
       - db.r4.16xlarge
 
   EnvironmentName:
-    Description: Environment name - dev or prod.
+    Description: Environment name - dev or prod
     Type: String
     Default: dev
     AllowedValues:
       - dev
       - prod
-    ConstraintDescription: Specify either dev or prod.
+    ConstraintDescription: Specify either dev or prod
+
+  # The database alarm configuration, currently not supported for Aurora PostgreSQL
+  DatabaseAlarmMaxCpuPercent:
+    Description: Database CPU % max for alarm (currently, Aurora MySQL only)
+    Type: Number
+    Default: 80
+    MinValue: 1
+    MaxValue: 99
+    ConstraintDescription: Must be a percentage between 1-99%
+
+  DatabaseAlarmReadLatencyMaxSeconds:
+    Description: Read latency max for alarm (currently, Aurora MySQL only)
+    Type: Number
+    Default: 1
+    MinValue: 1
+    ConstraintDescription: Must at least one
+
+  DatabaseAlarmWriteLatencyMaxSeconds:
+    Description: Write latency max for alarm (currently, Aurora MySQL only)
+    Type: Number
+    Default: 1
+    MinValue: 1
+    ConstraintDescription: Must at least one
+
+  DatabaseAlarmEvaluationPeriods:
+    Description: The number of periods over which data is compared to the specified threshold (currently, Aurora MySQL only)
+    Type: Number
+    Default: 1
+    MinValue: 1
+    ConstraintDescription: Must be at least one
+
+  DatabaseAlarmEvaluationPeriodSeconds:
+    Description: The time over which the specified statistic is applied. Specify time in seconds, in multiples of 60. Enhanced monitoring must be enabled if less than 500 seconds (currently, Aurora MySQL only)
+    Type: Number
+    Default: 300
+    MinValue: 60
+    ConstraintDescription: Must be at least 60 seconds
+
+  EnhancedMonitoring:
+    Default: false
+    Type: String
+    Description: The optional flag for enhanced monitoring (additional charges apply - https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Monitoring.OS.html) (currently, Aurora MySQL only)
+    ConstraintDescription: Only true or false are allowed
+    AllowedValues:
+      - true
+      - false
+
+  # Default is 200 MB
+  DatabaseAlarmSwapUsageInBytes:
+    Default: 209715200
+    Type: Number
+    Description: Number of swap usage bytes for alarm (if enabled - (Aurora MySQL only)
+    MinValue: 1
```

Comment:
I am not entirely sure. For most DBs, any swap usage is really bad. That said, there may be batch jobs that spike this up a bit, and I am trying to avoid a noisy alert. I looked at some active DBs and they all had zero to  ~ 10 MB of swap usage. Users can adjust themselves, but I am going to add a ticket to research further.